omaha/omaha_test.go

```diff
@@ -1,137 +0,0 @@
-package omaha
-
-import (
-	"encoding/xml"
-	"fmt"
-	"io/ioutil"
-	"os"
-	"testing"
-)
-
-func TestOmahaRequestUpdateCheck(t *testing.T) {
-	file, err := os.Open("../fixtures/update-engine/update/request.xml")
```

Comment:
This is partially an artifact of the code's previous life in mantle but I kept it because I'd rather have these in-line in code/documentation than as stand-alone files. I'll have more documentation coming soon and would like that to replace the pile of random fixtures.

So in short, just personal preference.